cmd/pod-scaler/frontend/node_modules/.bin/acorn

```diff
@@ -0,0 +1 @@
+../acorn/bin/acorn
```

Comment:
What is this?

Reply:
WTF, should have been omitted - it's part of the frontend